yt_dlp/extractor/youtube.py

```diff
@@ -3640,10 +3659,10 @@ def _extract_from_tabs(self, item_id, ytcfg, data, tabs):
             'uploader_url': channel_url,
             'thumbnails': thumbnails,
             'tags': tags,
+            'view_count': view_count,
+            'availability': self._extract_availability(data),
+            'last_updated_date': strftime_or_none(last_updated_unix, "%Y%m%d")
```

Comment:
TODO: create a new field of some kind for the date/timestamp of when video/playlist is updated on the site